owncloudTestUtil/build.gradle

```
@@ -21,7 +21,7 @@ android {
 dependencies {
     implementation project(':owncloudDomain')
     implementation project(':owncloud-android-library:owncloudComLibrary')
```

Try to group things with sense:

```suggestion
    implementation project(':owncloudDomain')
    implementation project(':owncloud-android-library:owncloudComLibrary')
    implementation project(':owncloudData')
    
    implementation libs.kotlin.stdlib
    ...
```

---

I think I have sorted better.